### ISSUE TITLE
SQS (QueueMessagingTemplate) still doesn't seem to use the application ObjectMapper to deserialise messages

### ISSUE THREAD

**Issue by brendanhatton** [awspring/spring-cloud-aws#119]
**Type**: Bug

**Component**: SQS

**Describe the bug**
The release notes for Spring Cloud AWS 2.3 describe a fix around using the existing ObjectMapper, if one is available in the context - however in my experience this doesn't seem to be the case. 

**Sample**
I have created a simple project which shows the problem and a workaround here: https://github.com/brendanhatton/spring-cloud-aws-messaging-bug

To summarise the project, the default QueueMessagingTemplate configuration will use a default ObjectMapper, which does not have the Kotlin module registered. A workaround is to manually set the object mapper as a message converter in the QueueMessagingTemplate - but I think this is expected to be the default behaviour.

**Comment by maciejwalkowiak** [comment]
Thanks for reporting. Indeed this can be improved

**Comment by internetstaff** [comment]
FWIW, `NotificationMessagingTemplate` suffers from the same issue. Odd, since both seemed to use the context's ObjectMapper before 2.3.

**Comment by internetstaff** [comment]
@maciejwalkowiak did this happen to fix NotificationMessagingTemplate as well?
Thanks!
